test/integration_test.py

```diff
@@ -264,7 +258,8 @@ def test_claw(self):
     def test_cosmo(self):
         spack_spec('cosmo')
         spack_spec('cosmo cosmo_target=gpu ~cppdycore')
-        spack_spec('cosmo cosmo_target=gpu +serialize +claw +zlib_ng +oasis')
+        #spack_spec('cosmo cosmo_target=gpu +serialize +claw +zlib_ng +oasis')
```

Comment:
Remove, reactivate or add a note why it's commented out.

Reply:
+serialize triggers patches. Would be nice to test them.
+serialize causes an additional dependency on serialbox. Would be nice to test this.

Reply:
There was a problem during concrezitation, but COSMO reaches end of life, therefore no further resources are invested.

Reply:
Since this is a known defect, I think we should do one of the following things:
- Remove the variant `serialize` from cosmo.
- Add a warning to the README.md, like "Warning: cosmo +serialize is broken"
- Create an issue for it.
It's only fair to propagate this information, otherwise someone might waist time on it.